src/components/NFTBook/SpecTableItemDateReleased.vue

```diff
@@ -12,7 +12,7 @@ export default {
       default: 'li',
     },
     date: {
-      type: String,
+      type: [String, Number],
```

Comment:
In what case date would be Number?

Reply:
timestamp from the [API](https://api.like.co/likernft/collection/col_book_931bb5a0-39e1-472e-963c-d542c9902ab4)